### ISSUE TITLE
Maven can't resolve some dependencies

### ISSUE THREAD

**Issue by Sponges** [DV8FromTheWorld/JDA#25]
In the latest version maven is unable to resolve jflac and tritonus-dsp.
```xml
<repositories>
    <repository>
        <id>letece</id>
        <url>http://lutece.paris.fr/nexus/content/repositories/lutece_third_party</url>
    </repository>
    <repository>
        <id>DV8FromTheWorld</id>
        <url>https://github.com/DV8FromTheWorld/Maven-Repository/raw/master/repo</url>
    </repository>
</repositories>
```
Manually adding the repositories seems to fix it. As far as I know this isn't an issue on my side.

**Comment by DV8FromTheWorld** [comment]
Interesting; It appears that the Gradle script isn't adding the <repositories> tags to the pom when it creates it.

**Comment by DV8FromTheWorld** [comment]
Okay so, when 1.3 is released to Maven Central, this problem will be fixed FOR MAVEN USERS ONLY. 
Unfortunately, this problem will persist for Gradle users. The reason for this is because Gradle does not respect the `<repositories> ... </repositories>` tag in the POM file. This is a known bug in Gradle.
https://issues.gradle.org/browse/GRADLE-611

Until Gradle implements support for the <repositories> tag, Gradle users will need to manually add JDA's required repositories to their `build.gradle` script

```
repositories {
    mavenCentral()

    //Used for tritonus:tritonus-dsp:0.3.6
    maven {
        url "http://lutece.paris.fr/nexus/content/repositories/lutece_third_party/"
    }

    //Used for
    // 'net.sourceforge.jaadec:jaad:0.8.4'
    // 'jflac:jflac:1.3'
    maven {
        url "https://github.com/DV8FromTheWorld/Maven-Repository/raw/master/repo"
    }
}
```

Furthermore, until another master release is released, Maven users will also need to manually define the repositories as @Sponges has done above.
When another master release is released (presumably v1.3), Maven users will no longer need to define the repositories.
